### DISCUSSION TITLE
Herrar e Umano

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#283] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/herrar-e-umano/

**Comment by one-eyed-willy** [comment]
Ah, não! Tudo menos miguxês! É demais até pra mim!

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [terça-feira, 22 de março de 2022](<https://www.vish.blog.br/imagens/tirinhas/herrar-e-umano/#comment_1075293348>):

> Ah, não! Tudo menos miguxês! É demais até pra mim!

I qTU Au nEU-MiGUxXxEixXx??!?! TodAxXx aKeLAxXx maIuScUlaxXx I MInUsCUlAxXx juNtAxXx i U ExXxCEXXu di exXxclAMaXXoExXx DExXxam U tExXxtu FAN-TAixXx-tI-ku!!!!!!!!!!!!!!!

**Comment by tali-zorah** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [terça-feira, 22 de março de 2022](<https://www.vish.blog.br/imagens/tirinhas/herrar-e-umano/#comment_1075306926>):

> I qTU Au nEU-MiGUxXxEixXx??!?! TodAxXx aKeLAxXx maIuScUlaxXx I MInUsCUlAxXx juNtAxXx i U ExXxCEXXu di exXxclAMaXXoExXx DExXxam U tExXxtu FAN-TAixXx-tI-ku!!!!!!!!!!!!!!!

*Keelah!* Pelo jeito encontramos uma orkuteira raiz por aqui!

**Comment by brunoaugusto** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [terça-feira, 22 de março de 2022](<https://www.vish.blog.br/imagens/tirinhas/herrar-e-umano/#comment_1075306926>):

> I qTU Au nEU-MiGUxXxEixXx??!?! TodAxXx aKeLAxXx maIuScUlaxXx I MInUsCUlAxXx juNtAxXx i U ExXxCEXXu di exXxclAMaXXoExXx DExXxam U tExXxtu FAN-TAixXx-tI-ku!!!!!!!!!!!!!!!

Credo! Como que alguém consegue ler uma coisa dessas sem tomar um Dramin antes?

**Comment by one-eyed-willy** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [terça-feira, 22 de março de 2022](<https://www.vish.blog.br/imagens/tirinhas/herrar-e-umano/#comment_1075306926>):

> I qTU Au nEU-MiGUxXxEixXx??!?! TodAxXx aKeLAxXx maIuScUlaxXx I MInUsCUlAxXx juNtAxXx i U ExXxCEXXu di exXxclAMaXXoExXx DExXxam U tExXxtu FAN-TAixXx-tI-ku!!!!!!!!!!!!!!!

Por quê eu fui sair da minha caverna...
